package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@exmg/exmg-paper-token-input",
   "flat": true,
-  "version": "3.0.3",
+  "version": "4.0.0",
```

Comment:
we should increase version via command npm version after merging to master

Reply:
done